data/lang/equipment-core/ar.json

```diff
@@ -187,6 +187,10 @@
     "description": "Name for the 'Fuel Scoop' equipment hardpoint",
     "message": "Fuel Scoop"
   },
+  "HARDPOINT_MINING_LASER": {
```

Comment:
Probably shouldn't be modifying the `ar` language directly...

Reply:
Unintentional.. moved to `en` in a fixup commit.